GameDev/ReadMe.md

```diff
@@ -1,6 +1,123 @@
-## Directions:
-1. In the folders for each week, update the ReadMe.md to contain a broad overview of the week's content.
-2. In the "livestream" folder (within each week's folder), paste in your completed script for that week's livestream.
-3. In that week's "homework" folder, put the solutions in the "solution" folder, and the starter code in the "starter" section of the "issues" folder. You should also include a .md file in the "issues" folder with directions for the homework.
+# Bitcamp GameDev
+
+Who doesn’t love video games? From Pac-Man to Mario, video games have ingrained themselves as a staple in day to day live - but how are they made? In the 80s, developers would simply have to code their games from scratch, but nowadays, there are a variety of game-development engines to automate the process. Unity is one of these engines; an industry standard used by multimillion dollar companies and small-time hobbyists alike. Beginner friendly and incredibly versatile, Unity is the gold standard for a reason, and learning it is a surefire way to turn yourself into a bona-fide game developer! 
+
+After learning the fundamentals of how to implement various features and tools in Unity, students will spend a few weeks coding their own games/tech demos to demonstrate what they’ve learned. 
+
+### Prerequisites
+
+Basic programming knowledge (C# is used by Unity, but knowing it isn’t necessary)
+
+### Agenda
+
+Introduction: Week 0 will be an introduction to GameDev, and a broad overview of what exactly makes a game fun! 
+The meat and potatoes: Weeks 1 through 3 will cover the barebone essentials of Unity, while weeks 4 through 7 will delve into more advanced usages.
+Finale: From week 8 to the end of the program, students will be working with their mentors to focus on their games and make sure they’re as good as can be for the final presentation!
+
+
+### Tools
+
+Unity Personal Edition
+Unity Asset Store
+
+## Curriculum Outline
+### Week 1
+
```

Comment:
Do you guys have a sample project? If you could add some gifs/images under each week that would be great! This would give others an idea of what a project they build in the camp could look like and show the progress throughout the weeks. The images could show how the students would gradually build the game.

Reply:
we can probably update this with excerpts from the lessons, since we don't have a sample project besides that (or maybe the homework but we'd rather not give out hints accidentally!)

Reply:
That's ok then, don't worry about the pictures for now 👍